manifests/07_deployment.yaml

```diff
@@ -75,7 +75,7 @@ spec:
 
       containers:
         - name: ovirt-csi-driver-operator
-          image: quay.io/openshift/csi-driver-operator
+          image: quay.io/openshift/origin-ovirt-csi-driver-operator:latest
```

Comment:
Just to make sure we are in the same page:  the files under `./manifests` directory are only here for reference. This driver is installed automatically by CSO, and when it's installed, CSO uses the manifests from this directory: https://github.com/openshift/cluster-storage-operator/tree/master/assets/csidriveroperators/ovirt.

Anyway, it's a good thing to keep the manifests in this repo updated as well, so lgtm.

Reply:
yes, this is just for reference, jthadden needed to install the operator on 4.5 where CSO doesn't handle this operator. Since the manual installation was successful, I updated the manifests and README as it may be beneficial to others

Reply:
Ack!

Note that at this stage of OCP 4.7 development, the bot requires a valid BZ to merge the PR. You can either create a BZ or wait until code-freeze is lifted (I believe in 1-2 weeks).